packages/api/test/index.test.ts

```diff
@@ -67,9 +67,9 @@ describe('api', function () {
         'head',
         'patch',
         'trace',
-        'list-data-sets',
-        'list-searchable-fields',
-        'perform-search',
+        'listDataSets',
```

Comment:
This is part of the work that the `camelCase` option in `getOperationId()` does by making something that can't be used as a JS accessor without arrayifying it (eg. `sdk['list-data-sets']()`) into the **much** cleaner `sdk.listDataSets()`.